bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -262,32 +273,32 @@ def main():
 
     perf_stats: PerfStatCollection = setup_perf_stats()
```

Comment:
How about we only set up stats that are mentioned in either the argument or the file? Then we won't have to filter the stats and we would generate them even if they are zero? [Here](https://mongodb.slack.com/archives/C02L31LDX5F/p1636332062018000)

Or should we do this later?

Reply:
Yep, I like it :)
I gave the change a quick attempt but we'll need to make larger changes to get this to work with batch_files containing multiple configs. I think this'll need to be separate ticket

Reply:
This is now tracked in [WT-8369](https://jira.mongodb.org/browse/WT-8369)